### ISSUE TITLE
[Bug][A11y]: NormalPeoplePicker do not announce the suggestionsAvailableAlertText in Andriod 15/16.

### ISSUE THREAD

**Issue by buzhouke** [microsoft/fluentui#35415]
### Package

azure-themes

### Package version

8.119.2 and 8.125.0

### React version

18

### Environment

```shell
Andriod 15/Andriod 16

"@fluentui/react": "^8.119.2"
"react": "18"
```

### Current Behavior

Talkback doesn't announce the suggestions list information after the People selection update list refresh. Currently Talkback remains silent.

This bug only happen in Andriod 15/Andriod 16, in IOS or Andriod 12, talkback could annouce the "suggestionsAvailableAlertText "

### Expected Behavior

Talkback should announce the suggestions list information after the People selection update list refresh. The Talkback should announce the value provided in "suggestionsAvailableAlertText"

### Reproduction

https://codepen.io/GAOBs-Gaib/pen/dPGjLvE

### Steps to reproduce

Open this page on an Android device （Must be Android 15 or  Android 16 ） https://codepen.io/GAOBs-Gaib/pen/dPGjLvE
Enable TalkBack accessibility service
Tap the input field below and type any character (e.g., "a")
Expected behavior: TalkBack should announce "Suggestions available"
Actual behavior: TalkBack does not announce suggestionsAvailableAlertText

### Are you reporting an Accessibility issue?

yes

### Suggested severity

Urgent - No workaround and Products/sites are affected

### Products/sites affected

Teams Updates App

### Are you willing to submit a PR to fix?

no

### Validations

- [x] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [x] The provided reproduction is a minimal reproducible example of the bug.
